sql/src/main/java/io/crate/operation/reference/information/InformationTablePartitionsExpression.java

```diff
@@ -30,6 +30,8 @@
 abstract class InformationTablePartitionsExpression<T>
     extends RowContextCollectorExpression<PartitionInfo, T> {
 
+    public static final BytesRef tableType = new BytesRef("BASE TABLE");
```

Comment:
IMO the table type should be defined as part of the TableInfo, @mfussenegger what do you think?
how many table types are there?
if that is too much overhead, I would define the static variable inside the expression https://github.com/crate/crate/pull/5641/files#diff-1f715bb8929ec4f8885636b46069d228R156